__tests__/news/news.query.test.js

```diff
@@ -1,20 +1,20 @@
 /* eslint-disable no-undef */
+/* eslint-disable no-undef */
+require('dotenv').config();
```

Comment:
Please, remove this line, it is going to use env config for development. Why have you added this line?

Reply:
Fixed